aiida/tools/importexport/common/archive.py

```diff
@@ -61,7 +61,7 @@ def extract_zip(infile, folder, nodes_export_subfolder=None, check_files=('data.
         incorrect formats
     """
     warnings.warn(
-        'extract_zip function is deprecated and will be removed in AiiDA v2.0.0, use `safe_extract_zip` instead',
+        'extract_zip function is deprecated and will be removed in AiiDA v2.0.0, use the archive-path package instead',
```

Comment:
TBH they can do whatever they want, they just can't use this function lol. `export` does not offer the same, safe by default (no paths accessed outside of the archive, no symlinks/devices) or progress callback that this function does.

Reply:
the "function" in question is the `extract_tree` method. But I don't think it would be useful to specify this, in case it changes in the future.
(I imagine it would be very rare anyway that someone is currently using this function)

Reply:
Thanks - I think my original suggestion was partly a misunderstanding on my part that `safe_extract_zip` was no longer in aiida-core.

I think the main question is whether we want `safe_extract_zip` to be part of the public API (independent of nesting depth) - if yes, then your suggestion is fine, if not, we should suggest to use something else.

Reply:
> I think the main question is whether we want safe_extract_zip to be part of the public API

I bit confused by this, `safe_extract_zip` is now removed from aiida-core so no it won't be part of the API?
Note it was added and removed before a release was made so no deprecation needed

Reply:
Ok, so I was right after all (sorry, perhaps I checked out the wrong branch locally to check) - well, then you should certainly adapt the warning

Reply:
>  well, then you should certainly adapt the warning

you mean change what it is now or keep as is?

Reply:
point to extract_tree in archive-path in 98f0b9d